packages/pn-commons/src/components/__test__/PnAutocomplete.test.tsx

```diff
@@ -14,7 +14,7 @@ describe('PnAutocomplete Component', () => {
       />
     );
     expect(result.container).toHaveTextContent(/Input label/i);
-    const button = result.queryByTitle('Open');
+    const button = result.queryByTitle('');
```

Comment:
change this row. we can use result.container.querySelector('button[class^="MuiAutocomplete-popupIndicator"]')

Reply:
solved with: `result.container.querySelector('.MuiAutocomplete-popupIndicator')`  because the solution proposed couldn't find the dom element(button).